test/integration/auth_webhook_test.go

```diff
@@ -188,4 +190,111 @@ func TestAuthWebhook(t *testing.T) {
 		err = cli.Activate(ctx)
 		assert.Equal(t, codes.Unauthenticated, status.Convert(err).Code())
 	})
+
+	t.Run("authorized request cache test", func(t *testing.T) {
+		reqCnt := 0
+		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			req, err := types.NewAuthWebhookRequest(r.Body)
+			assert.NoError(t, err)
+
+			var res types.AuthWebhookResponse
+			res.Allowed = true
+
+			_, err = res.Write(w)
+			assert.NoError(t, err)
+
+			if req.Method == types.PushPull {
+				reqCnt++
+			}
+		}))
+
+		var authorizedTTL uint64 = 2
+		conf := helper.TestConfig(server.URL)
+		conf.Backend.AuthorizationWebhookCacheAuthorizedTTLSec = authorizedTTL
+
+		agent, err := yorkie.New(conf)
+		assert.NoError(t, err)
+		assert.NoError(t, agent.Start())
+		defer func() { assert.NoError(t, agent.Shutdown(true)) }()
+
+		ctx := context.Background()
+		cli, err := client.Dial(agent.RPCAddr(), client.Option{Token: "token"})
+		assert.NoError(t, err)
+		defer func() { assert.NoError(t, cli.Close()) }()
+
+		err = cli.Activate(ctx)
+		assert.NoError(t, err)
+
+		doc := document.New(helper.Collection, t.Name())
+		err = cli.Attach(ctx, doc)
+		assert.NoError(t, err)
+
+		// request(PushPull) 3
+		for i := 0; i < 3; i++ {
+			doc.Update(func(root *proxy.ObjectProxy) error {
+				root.SetNewObject("k1")
+				return nil
+			})
+			cli.Sync(ctx)
+		}
+
+		time.Sleep(time.Duration(authorizedTTL) * time.Second)
+
+		for i := 0; i < 3; i++ {
+			doc.Update(func(root *proxy.ObjectProxy) error {
+				root.SetNewObject("k1")
+				return nil
+			})
+			cli.Sync(ctx)
+		}
+
+		assert.Equal(t, reqCnt, 2)
```

Comment:
This test seems to contain constants related to the value of authorizedTTL. How about expressing the value of this constant based on authorizedTTL(e.g. `3` -> `authorizedTTL + 1`)?

Reply:
Thank you so much for your comments.
I didn't understand your intentions. Could you please explain a little more?

The reason for requesting requests 3 times was intended to be multiple requests.
I didn't think the constant 3 had anything to do with the authorizedTTL value.

Reply:
1. 2 seconds to 1 second to avoid increasing test time
2. We need to switch exected and actual in assert.Equal
3. More detailed explanation in the comments

```suggestion
		var authorizedTTLSec uint64 = 1
		conf := helper.TestConfig(server.URL)
		conf.Backend.AuthorizationWebhookCacheAuthorizedTTLSec = authorizedTTLSec

		agent, err := yorkie.New(conf)
		assert.NoError(t, err)
		assert.NoError(t, agent.Start())
		defer func() { assert.NoError(t, agent.Shutdown(true)) }()

		ctx := context.Background()
		cli, err := client.Dial(agent.RPCAddr(), client.Option{Token: "token"})
		assert.NoError(t, err)
		defer func() { assert.NoError(t, cli.Close()) }()

		err = cli.Activate(ctx)
		assert.NoError(t, err)

		doc := document.New(helper.Collection, t.Name())
		err = cli.Attach(ctx, doc)
		assert.NoError(t, err)

		// 01. multiple requests to update the document.
		for i := 0; i < 3; i++ {
			assert.NoError(t, doc.Update(func(root *proxy.ObjectProxy) error {
				root.SetNewObject("k1")
				return nil
			}))
			assert.NoError(t, cli.Sync(ctx))
		}
		assert.Equal(t, 1, reqCnt)

		// 02. multiple requests to update the document after eviction by ttl.
		time.Sleep(time.Duration(authorizedTTLSec) * time.Second)
		for i := 0; i < 3; i++ {
			doc.Update(func(root *proxy.ObjectProxy) error {
				root.SetNewObject("k1")
				return nil
			})
			cli.Sync(ctx)
		}
		assert.Equal(t, 2, reqCnt)
```

Reply:
@hackerwins Thank you so much for the detailed explanation! It's corrected :)